sunpy/map/tests/test_mapbase.py

```diff
@@ -890,7 +890,7 @@ def test_bad_coordframe_repr(generic_map):
     generic_map.meta['CTYPE2'] = "STUART2"
     with pytest.warns(UserWarning,
                       match="Could not determine coordinate frame from map metadata"):
-        assert 'Unknown' in generic_map.__repr__()
+        assert 'Unknown' in generic_map.__str__()
```

Comment:
I don't think we should remove the test for `__repr__`. There should just be an additional test for `__str__`